R/mail.R

```diff
@@ -9,40 +9,51 @@
 
 #' Creates fake mbox data
 #'
+#' Used to create a fake mbox with the specified parameters and returns the output to the console
+#'
 #' @param mlist specific mailing list associated with this mbox data
-#' @param reply_from email of sender
-#' @param reply_to email of receiver
-#' @param reply_cc carbon copy of email to additional recipients
-#' @param reply_datetime Date, time in the following format: 2023-02-11T09:30:00
-#' @param timezone Abbreviated timezone of the reply eg: HST or EST
-#' @param reply_subject subject of the email
-#' @param reply_body the body of the email
+#' @param reply_from_author name of sender. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_from_email email of sender. eg: 'sender@gmail.com'
+#' @param reply_to_author name of receiver. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_to_email email of receiver eg: 'receiver@gmail.com'
+#' @param reply_cc_author name of carbon copy of email to additional recipients. Formatted as: FIRSTNAME LASTNAME
+#' @param reply_cc_email email of carbon copy recipients
+#' @param reply_datetime Date, time in the following format: 2023-02-11T09:30:00  Be sure to add the capital T to separate date and time
+#' @param timezone Abbreviated timezone of the reply eg: HST
```

Comment:
Do you know if there is any reference for the timezone IDs in the posix function we can link here? For example, people on daylight savings may type PDT. Not sure if that would work or not. Or some may have to type in GMT, etc.